assets/js/modules/adsense/components/module/ModuleOverviewWidget/Stats.js

```diff
@@ -74,6 +84,7 @@ const Stats = ( {
 				color: '#616161',
 				fontSize: 12,
 			},
+			ticks: dates,
```

Comment:
This option is duplicated across the three graphs

I could actually move this logic into `GoogleChart`and have `GoogleChart` creates the `hAxis.ticks` option from its `data` prop

Reply:
No need to move it. It looks good to me.